stackit/internal/services/logme/instance/resource.go

```diff
@@ -242,6 +413,8 @@ func (r *instanceResource) Create(ctx context.Context, req resource.CreateReques
 		if resp.Diagnostics.HasError() {
 			return
 		}
+	} else {
+		parameters = nil
```

Comment:
Why is this needed? And if so, couldn't this be avoided with a `var parameters *parametersModel` declaration above? Then create a parametersModel struct inside the if

Reply:
I see that this being nil is useful to check in the `toInstanceParams`

Reply:
Good suggestion, applied it